README.md

```diff
@@ -1,2 +1,8 @@
 # Synergile
  This bot is a project engineered by CodeWritten, with help from Jedi3, Wakfi, and Napkins
+
+# Where stuff is
+ If you are looking for most of the commands, find them in the commands folder, If you are looking for the utilities, find them in the util folder. Of course, the main file for the bot is bot.py
```

Comment:
I think anyone with basic reading comprehension could figure that much out since that's the point of self-descriptive naming (except maybe util but that's still pretty clear). The goal of additional description in the README would be to (in no particular order):

a. Describe the project, purpose, goals, progress, and anything else that fits this feeling.
b. Give some specific points of interest within the sub folders or otherwise describe how to interact with the project.
c. Indicate how to report a bug, report a security vulnerability, or request a feature, for those who are here for any of those reasons.

It would also be good to point prospective contributors towards the issues tab, perhaps the `good-first-issue` tag or whatever we end up using as an equivalent to that concept.